_posts/2025-06-30-pyos-scipy-2025.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
At the top of every blog post we have a yaml section that contains metadata about the post. This might be easier to work on in hackmd.io so i created a file there for you to look at. 

https://hackmd.io/@pyopensci/HyNpu0WBex